### PR TITLE
launch week 5 blog posts

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

* adds php documentation
* fixes docs for c# not being linked from overview page
* adds next.js sourcemap blog post
* adds @vercel/otel blog post
* adds clickhouse performance blog post

## How did you test this change?

[vercel ci](https://highlight-landing-3n4wjxk0a-highlight-run.vercel.app/)

## Are there any deployment considerations?

no

## Does this work require review from our design team?

no

### CODE REVIEW

File: blog-content/lw5-clickhouse-performance-optimization.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+CPU Usage during merges can be incurred by the conversion of parts from compact to wide. When data is inserted
+compressed, it must be decompressed into the wide format for it to be merged. If inserting large batch sizes (100k+ rows
+at 10MB+ data), a table level setting that helps is using min_rows_for_wide_part=0, min_bytes_for_wide_part=0 to make
+sure that ClickHouse will keep inserted data as WIDE to avoid having to convert back and forth between the compact and
+narrow format, since each conversion incurs a CPU cost.
+
+### Optimizing Order By Granularity
+
+TODO what does the ORDER BY do
```
Comment (Vadman97) [inline blog-content/lw5-clickhouse-performance-optimization.md:78]: still WIP

File: blog-content/lw5-d3-metrics.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Day 3: Metrics & APM"
-createdAt: 2024-02-02T12:00:00Z
+createdAt: 2024-04-29T12:00:00Z
```
Comment (jay-khatri) [inline blog-content/lw5-d3-metrics.md:3]: can we keep everything at/after day 3 as older? that way they're essentially hidden.

File: blog-content/lw5-clickhouse-performance-optimization.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+```SQL
+select partition, sum(num_parts)
+from clusterAllReplicas(default, system.merges)
+where table = 'my_table'
+group by 1
+order by 2 desc;
+```
+
+system drop parts
```
Comment (jay-khatri) [inline blog-content/lw5-clickhouse-performance-optimization.md:182]: is this finished?

File: blog-content/lw5-vercel-otel-nextjs-tracing.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+                'highlight.session_id': session_id,
+                'highlight.request_id': request_id,
+            }
+        },
+    })
+}
+
+```
+
+## Visualizing Edge Runtime Spansin Highlight.io
```
Comment (jay-khatri) [inline blog-content/lw5-vercel-otel-nextjs-tracing.md:146]:
```suggestion
## Visualizing Edge Runtime Spans in Highlight.io
```

File: blog-content/lw5-clickhouse-performance-optimization.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+authorPFP: 'https://lh3.googleusercontent.com/a-/AOh14Gh1k7XsVMGxHMLJZ7qesyddqn1y4EKjfbodEYiY=s96-c'
+tags: Launch Week 5
+metaTitle: "Optimizing Clickhouse Cost + CPU: 5 Tactics That Worked"
+---
+
+Scaling an observability stack isn’t an easy problem. At Highlight.io, we ingest nearly 100 terabytes of data from our
+customers every month. A large chunk of the volume goes to our ClickHouse cluster, handling our client’s Logs, Traces,
+and Session metadata. Not only do we need to gracefully handle the volume and daily spikes in the traffic patterns, but
+we also need to ensure that the data is ingested in an efficient format that is performant to query.
+
```
Comment (jay-khatri) [inline blog-content/lw5-clickhouse-performance-optimization.md:21]:
```suggestion
About a year ago, once we started onboarding larger customers, we started hitting scaling issues with our Clickhouse set up that warranted several "phases" of improvements to our query and ingest systems. This post goes through the initial realization that we had a challenge on our hands, and how we tackled it.
```

File: blog-content/lw5-clickhouse-performance-optimization.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+## The Incident
+
+As our data ingest has grown, we have needed to make improvements to our ClickHouse configuration to keep up and further
+improve query performance. For example, one day, we onboarded a large customer that sends close to 1 billion trace spans
+a day, adding another terabyte to our data ingestion. Beyond the total volume, the larger problem was in the number of
```
Comment (jay-khatri) [inline blog-content/lw5-clickhouse-performance-optimization.md:26]:
```suggestion
One day, we onboarded a large customer that sent close to 1 billion trace spans
a day, adding another terabyte to our data ingestion. Beyond the total volume, the larger problem was in the number of
```

File: blog-content/lw5-clickhouse-performance-optimization.md [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+
+Unlike traditional OTLP SQL, the way that data is written to a ClickHouse table affects query performance.
+For instance, if data is written in smaller chunks, ClickHouse ends up creating more “parts” (in other words, files) to
+store it. When querying, ClickHouse has to merge and possibly reorder the results from all relevant parts. Additionally,
+ClickHouse will asynchronously merge data as it is initially ingested to get it into the right format for long term
+storage, but this process consumes CPU on background threads that ultimately may decrease performance of concurrent
+storage. Inserting data with the right configuration helps the merges that ClickHouse has to perform and can speed up
+how much each merge takes.
+
+## Optimizing CPU Usage: Our Strategies
```
Comment (jay-khatri) [inline blog-content/lw5-clickhouse-performance-optimization.md:49]:
```suggestion
```

**Review by jay-khatri — APPROVED** [review]

**Pull request merged**